src/PhpUnit/SymfonyAssertsTrait.php

```diff
@@ -71,8 +72,9 @@ public function assertRequestMatch(
         SchemaManager $schemaManager,
         $message = ''
     ) {
-        $path = $request->getRequestUri();
+        $path = $request->getPathInfo();
```

Comment:
In Symfony the requestUri contains the QueryString. So if you have a querystring you won't match any path.